### ISSUE TITLE
pylint-1.5.4 warns about unused-argument with __div__

### ISSUE THREAD

**Issue by bcl** [PyCQA/pylint#807]
The following:

```python
class Size(object):
    def __div__(self, other, context=None):
        return other / 10

    def __add__(self, other, context=None):
        return other + 10
```

Doesn't complain about unused_argument in __add__ but does warn about __div__

It appears to only complain about special methods that aren't part of the stdlib.

**Comment by PCManticore** [comment]
I think we're missing this method in pylint.checkers.utils._SPECIAL_METHODS_PARAMS. Do you mind sending a PR for adding it?
